pkg/startup/v7/data/worker/etc/mdsd.d/mdsd.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If you use the pattern above all files for masters and nodes could be re-used and not copy-paste. basically maintained once.